gsb_research_hub_subtheme.libraries.yml

```diff
@@ -96,13 +96,8 @@ airtable-list-builder:
   js:
     dist/js/airtable-list-builder.js: {}
     //cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js: {type: external, minified: true}
-#      dist/css/print.css: { media: print }
-#    base:
-#      dist/css/base.css: { weight: 100 }
-#    layout:
-#      dist/css/layout.css: {}
-#    component:
-#      dist/css/components.css: {}
-#    state:
-#      dist/css/state.css: {}
+    //cdnjs.cloudflare.com/ajax/libs/chosen/1.8.7/chosen.jquery.min.js: {type: external, minified: true}
```

Comment:
Why not always load over https?

Reply:
I think I did it this way because it's what the documentation showed as the way to do it.